frontend/src/metabase/query_builder/components/DataSelector/DataSelector.module.css

```diff
@@ -0,0 +1,45 @@
+.TableSearchContainer {
```

Comment:
I'm pretty sure our convention is to use `.camelCase`

Reply:
right, but I wanted to have one to one migration from StyledComponents to css modules so it's easier to review when the class name is the same as a component name was

Reply:
That's cool but I think it was not necessary :)